src/js/ReportMetricConfig.js

```diff
@@ -208,6 +209,11 @@ var ReportMetricConfig = function () {
         order(metricOrder);
     }
 
+    const setChecked = function (metricChecked) {
```

Comment:
I can use the same configuration for all metrics configuration.
`[{name: 'metric1', checked: true}, {name: 'metric2', checked: false}]`